### PR TITLE
Added currency to checkout token template

### PR DESCRIPTION
**Pull request by liquorvicar**
This is my first contribution to payum and I couldn't find any contribution guidelines so let me know if there's any styling issues or extra tests etc I need to add.

### CODE REVIEW

File: src/Payum/Stripe/Resources/views/Action/obtain_checkout_token.html.twig [hunk 1]
```diff
@@ -10,7 +10,8 @@
             data-image="{{ model.image|default("") }}"
             data-name="{{ model.name|default("") }}"
             data-description="{{ model.description|default("") }}"
-            data-amount="{{ model.amount }}">
+            data-amount="{{ model.amount }}"
+            data-currency="{{ model.currency }}">
```
Comment (makasim) [inline src/Payum/Stripe/Resources/views/Action/obtain_checkout_token.html.twig]: looks good, shouldn't we set usd as default here?

Comment (liquorvicar) [inline src/Payum/Stripe/Resources/views/Action/obtain_checkout_token.html.twig]: Good call. I wasn't sure whether the currency would always be set on the model or not but adding the default gets round that (and doesn't break existing behaviour). Have amended pull request.

**Comment by makasim** [comment]
Thanks.

**Comment by liquorvicar** [comment]
@makasim Thanks, any chance you can tag this as a release?

**Comment by makasim** [comment]
@liquorvicar I do not plan to release 1.0 in near future cuz there are some  issues which must be included it. What we can do is to backport the fix to 0.15 branch. It is stable one and I can tag at any moment

**Comment by liquorvicar** [comment]
Adding it to the 0.x version would be absolutely fine. Thanks!

**Pull request merged**